modules/apps/jaggery/mdm/src/mdm/units/platform-configuration/platform-configuration.hbs

```diff
@@ -472,8 +472,8 @@
                     <br>Please click <b>"Go back to configurations"</b>, if you wish to save another configuration or click
                     <b>"Exit"</b> to complete the process and go back to the dashboard.
                     <hr />
-                    <button class="wr-btn" onclick="window.location.href='/mdm/platform-configuration'">Go back to configurations</button>
-                    <button class="wr-btn" onclick="window.location.href='/mdm'">&nbsp;&nbsp;&nbsp;&nbsp;Exit&nbsp;&nbsp;&nbsp;&nbsp;</button>
+                    <button id=btnBack class="wr-btn" onclick="window.location.href='/mdm/platform-configuration'">Go back to configurations</button>
```

Comment:
unquoted attributes are valid on html5 specification, but as a best practice do recommend to use quotes.